### DISCUSSION TITLE
Deployment roll back feature

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#1611] [Unsupported Versions]
Hi,

There is a command to roll back your last deployment?
In cases where your customizations got broken by new version changes.

**Comment by TOoSmOotH** [comment]
There shouldn't be any customizations happening in the defaults directory. This is the code that gets replaced. What are you trying to customize?

**Reply by idolaman** [reply]
For example, in security onion 2.2, i added new logstash output in manager on local directory (salt and pillar), when using soup and deploying new 2.3 version, there was index name bugs (the date in index name specified twice) because of changes in #1506 and 69a04dedd33c6d80ff774403e0f29c49d46b90de
